.prettierrc.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   bracketSpacing: true,
+  singleQuote: false,
```

Comment:
Remove this. At  most of the places we have used single quotes. It will result into unncessary file changes.

Reply:
Done

Reply:
Done